cypress/integration/instructor.spec.js

```diff
@@ -0,0 +1,32 @@
+/// <reference types="cypress" />
+// Tests the functionality from the Instructor perspective
+
+describe("Instructors", () => {
+    before(() => {
+
+        cy.visit(Cypress.env('login_url'));
+        cy.get("[name=csrfmiddlewaretoken]")
+            .should("exist")
+            .should("have.attr", "value")
+            .as("csrfToken");
+
+        cy.get("@csrfToken").then((token) => {
+            cy.request({
+                method: "POST",
+                url: Cypress.env("login_url"),
+                form: true,
+                body: {
+                    username: "instructor",
+                    password: "1234567890",
+                },
+                headers: {
+                    "X-CSRFTOKEN": token,
+                },
+            });
+        });
+    })
```

Comment:
This login bit is copied in a bunch of places, perhaps extract to a shared function?